crates/sozo/ops/src/tests/migration.rs

```diff
@@ -240,6 +245,65 @@ async fn migrate_with_metadata() {
     }
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn migrate_with_auto_authorize() {
+    let config = build_full_test_config("../../../examples/spawn-and-move/Scarb.toml", false)
+        .unwrap_or_else(|c| panic!("Error loading config: {c:?}"));
+    let ws = setup_ws(&config);
+
+    let migration = setup_migration().unwrap();
+
+    let manifest_base = config.manifest_path().parent().unwrap();
+    let mut manifest =
+        BaseManifest::load_from_path(&manifest_base.join(MANIFESTS_DIR).join("dev").join(BASE_DIR))
+            .unwrap();
+
+    let overlay_manifest = OverlayManifest::load_from_path(
+        &manifest_base.join(MANIFESTS_DIR).join("dev").join(OVERLAYS_DIR),
+    )
+    .unwrap();
+
+    manifest.merge(overlay_manifest);
+
+    let sequencer =
```

Comment:
For the next PRs, once the Katana runner update is merged with Cairo 2.6, let's try to use Katana runner instead. For this one let's keep that. :)